helm/hpcc/templates/eclagent.yaml

```diff
@@ -67,7 +67,7 @@ data:
 {{- end }}
         spec:
           {{- include "hpcc.placementsByJobTargetType" (dict "root" .root "job" $appJobName "target" .me.name "type" "eclagent") | indent 10 }}
-          serviceAccountName: "hpcc-default"
+          serviceAccountName: "hpcc-agent"
```

Comment:
should probably be pushed to a separate JIRA/PR.
Hit this when running a test query that forced use of local DFS service from agent.
In that case, the agent needs a service account, to auto discover the local env. dfs service.